torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16266,14 +16266,14 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
                          device_type='cuda', dtypes=(torch.bfloat16, torch.float16, torch.float32),
                          active_if=TEST_WITH_ROCM and PLATFORM_SUPPORTS_MEM_EFF_ATTENTION),
             # for element 1, was torch.Size([4, 4, 0]) but real shape was torch.Size([16, 3, 0])
-            DecorateInfo(unittest.expectedFailure, "TestMeta", "test_dispatch_meta_outplace", device_type="cuda",
+            DecorateInfo(unittest.skip, "TestMeta", "test_dispatch_meta_outplace", device_type="cuda",
```

Comment:
Why are you replacing expectedFailure with skip here?

Reply:
We didn't understand why they were listed XFAIL in the first place, seems that we should list as skipped instead to move towards enabling these at some point. We can maintain XFAIL though if that is preferred here.

Reply:
Moved back to XFAIL for now.